data.sql

```diff
@@ -7,3 +7,17 @@ INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) V
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Pikachu','2021-01-07', 1, false, 15.04);
 
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Devimon','2017-05-12', 5, true, 11);
+
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Charmander','2020-02-8', 0, false, 11);
+
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Plantmon','2021-09-15', 2, true, 5.7);
+
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Squirtle','1993-04-02', 3, false, 12.13);
+
```

Comment:
- [ ] Kindly make the **"``Squirtle``"** animal has the data like the ones given on the **[project's requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md)** like on the `weight_kg` column. 
Once you update it make sure, you also update other `queries` required for the project to still follow the requirements.